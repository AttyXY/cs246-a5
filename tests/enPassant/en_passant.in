game human human
move a2 a4
move e7 e5
move a4 a5
move e5 e4
move d2 d4
move e4 f3
move e4 d3
