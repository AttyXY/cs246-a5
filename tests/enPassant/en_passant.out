WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  P-------
3  --------
2  -PPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  ----p---
4  P-------
3  --------
2  -PPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  P---p---
4  --------
3  --------
2  -PPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  P-------
4  ----p---
3  --------
2  -PPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  P-------
4  ---Pp---
3  --------
2  -PP-PPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppp-ppp
6  --------
5  P-------
4  ---Pp---
3  --------
2  -PP-PPPP
1  RNBQKBNR
   abcdefgh

Invalid move. Input another move.
8  rnbqkbnr
7  pppp-ppp
6  --------
5  P-------
4  --------
3  ---p----
2  -PP-PPPP
1  RNBQKBNR
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
